### ISSUE TITLE
Some knobs can't be turned in UI

### ISSUE THREAD

**Issue by yurivict** [sadko4u/lsp-plugins#247]
In the Carla host some knobs can't be properly turned, see the video screenshot below:

https://user-images.githubusercontent.com/271906/164335113-c0dbb665-8d49-4bcb-adcb-51e580914281.mp4

Please note how some knobs change numeric values without turning, which I think isn't expected.

Version 1.2.0
Carla-2.4.2_1
OS: FreeBSD 13

**Comment by sadko4u** [comment]
Hmm... Never had this issue... Need to check.
Maybe need to install FreeBSD 13 in virtual environment and verify the behaviour.

**Comment by sadko4u** [comment]
Also please note: there's a but in Sampler UI that I've recently fixed while moving towards 1.2.1.
Is it possible for you to try to checkout the `devel` branch from git, fetch all dependencies by `make fetch` and test the latest version from `devel`?
The usual build process looks like:
```
git clone git@github.com:sadko4u/lsp-plugins.git
cd lsp-plugins
git checkout -b devel origin/devel
make config
make fetch
make -j<N>
sudo make install
```

**Comment by sadko4u** [comment]
Hello!
Just compiled with Clang the `devel` branch of plugins.
Can not reproduce the problem with knobs related to Carla and LSP:
https://www.youtube.com/watch?v=wtFaZYjU8E0

Are you using XOrg server or Wayland?

**Comment by yurivict** [comment]
> Are you using XOrg server or Wayland?

I am using xorg and xfce4.

**Comment by sadko4u** [comment]
When running XFCE4, I found that `libxfce4powermanager` is eating up all the memory and swap and finally getting the XFCE crashing. Any ideas of curing this problem?

**Comment by sadko4u** [comment]
Finally I got XFCE4 running well. Seems that I was required to enable dbus and hald in `/etc/rc.conf`.
Can not reproduce the problem. All knobs work well and smoothen.

**Comment by sadko4u** [comment]
Just released 1.2.1. Please check if you can reproduce the problem on 1.2.1.

**Comment by yurivict** [comment]
> Just released 1.2.1. Please check if you can reproduce the problem on 1.2.1.

The problem is still there.

Knobs "Head cut", "Tail cut", "Fade in", "Fade out" are affected. When the mouse is clicked and moved the numbers change but there is no knob rotation.

**Comment by sadko4u** [comment]
Do you use `unclutter` or something similar?

**Comment by thoth-amon** [comment]
The problem is the knobs go from 0 to 64,000ms. When dealing with a sample that is only a few hundred ms this means the vast majority of the knob's turning circumference is irrelevant and the turning you can do within the sample is not visible.  I think this is giving the OP the sense that the knob is not actually turning. It is but just at a rate that can't be seen.

It would be better if these knobs instead of a set amount of ms were 0 to "the sample length in ms". The fixed ms also causes a bug in the loop section when the end ms is longer than the sample, the looping stops working.

**Comment by sadko4u** [comment]
We're working on the limits and probably once we'll do them well.

**Comment by sadko4u** [comment]
Fixed in 1.2.5
